app/api/helpers/mail.py

```diff
@@ -154,3 +177,11 @@ def send_email_after_event(email, event_name, upcoming_events):
             upcoming_events=upcoming_events
         )
     )
+
+
+def send_email_change_user_email(new_email, email):
+    s = get_serializer()
+    hash = urllib.quote(base64.b64encode(s.dumps([email, str_generator()])))
+    link = make_fe_url(path='/email/verify?token={token}'.format(token=hash))
```

Comment:
Could we make `make_fe_url` accept a dict of query params instead of like this ?

So usage would be like:

```python
link = make_fe_url('/url/without-query')
```

```python
link = make_fe_url('/url/with-query', {
  "token": "xyzzy"
})
```